.github/workflows/android.yml

```diff
@@ -83,3 +84,9 @@ jobs:
           groups: owner
           file: app/build/outputs/apk/release/app-release.apk
           releaseNotes: "OSS"
+
+      - name: Upload Artifact
```

Comment:
Don't want to really upload everything after each push. No need for that. Let's keep this PR about ktlint only : )